<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QDialog" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family:century gothic;
color:white;
}
QLabel{
font-size:16px;
}
QLineEdit{
font-size:16px;
font-weight:bold;
color:black;
background:white;
}
#changePassword{
background:#282828;
}
QPushButton {
font-size:15px;
font-weight:bold;
border:none;
border-radius:5px;
background: #ff0303;
}
#eye,#eye1,#eye2{
background:transaprent;
}
QPushButton:hover {
background:#49ebff;
color:black;
}</string>
  </property>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eye1">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>52</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/eyes.png</normaloff>:/eyes.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>38</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="eye">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>12</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/eyes.png</normaloff>:/eyes.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>38</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="oldPasswordText">
      <property name="text">
       <string>Enter Old Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="oldPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="eye2">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>92</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <disabledoff>:/eyes.png</disabledoff>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>38</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <property name="spacing">
     <number>21</number>
    </property>
    <item>
     <widget class="QLabel" name="newPasswordText">
      <property name="text">
       <string>Enter New Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <property name="spacing">
     <number>40</number>
    </property>
    <item>
     <widget class="QLabel" name="confirmPasswordText">
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="confirmPassword">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
